sqlalchemy/ormDemo.py

```diff
@@ -0,0 +1,120 @@
+# Example SQLAlchemy ORM program that uses the Ingres dialect.
+#
+# The code creates table "Employees" (drops/recreates if exists),
+#  adds rows to the table, updates a row, deletes a row,
+#  queries and prints the rows after each operation.
+#
+# Environment variables:
+#   SQLALCHEMY_INGRES_ODBC_DRIVER_NAME - Name of Ingres ODBC driver, required
+#   DEMO_DB - Name of database to use, required
+#   DEMO_ID - Userid for connecting, optional
+#   DEMO_PW - Password for connecting, optional
+#   Note: DEMO_ID and DEMO_PW will only be used if both are set.
+#
+# Code is partly modeled after the tutorial in video:
+#   "SQLAlchemy Turns Python Objects Into Database Entries"
+#    by NeuralNine at:  https://www.youtube.com/watch?v=AKQ3XEDI9Mw
+#
+# Code tested with these Python packages:
+#   SQLAlchemy        2.0.25
+#   ingres_sa_dialect 0.4
+#   pypyodbc          1.3.6
+#   setuptools        63.2.0
+
+import os, sys
```

Comment:
be nice to follow pep-8, please run this file through:

  * isort - https://pypi.org/project/isort/
  * black - https://pypi.org/project/black/

In the absence of a firm coding style policy, please adopt that for Python code. We'll probably want to relax a few options (Black has very few options, so that at least makes life easy, you put up it or don't use Black 😆 )

Reply:
Ran isort and black formatters.